build-system/replace-secrets.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
```

Comment:
This file is only ever executed by the Cloud Build system, never locally.

Reply:
Optional: I believe you can add a check to verify that this doesn't run locally by checking `process.env.NODE_ENV !== 'production'` (or `===` depending on what you want)

Reply:
Done.